packages/cashier/src/stores/general-store.js

```diff
@@ -72,6 +72,28 @@ export default class GeneralStore extends BaseStore {
         return this.is_p2p_visible && !this.root_store.client.is_eu;
     }
 
+    /**
+     * Gets the notifications from local storage, within p2p_settings, where it checks which notification has
+     * been seen. The number of unseen notifications is displayed in vertical tab, notifications count, for P2P.
+     *
+     * @returns {number} Notifications that have not been seen by the user.
+     */
+    @computed
+    get p2p_unseen_notifications() {
+        const p2p_settings = JSON.parse(localStorage.getItem('p2p_settings') || '{}');
+        const local_storage_settings = p2p_settings[this.root_store.client.loginid];
+
+        if (isEmptyObject(local_storage_settings)) {
+            return 0;
```

Comment:
Why 0?

Reply:
If there is no notifications, I'll return 0 so the notifications count is set to 0